packages/block-editor/src/components/use-moving-animation/index.js

```diff
@@ -139,10 +139,6 @@ function useMovingAnimation( { triggerAnimationOnChange, clientId } ) {
 		const y = Math.round( previous.top - destination.top );
 
 		controller.start( { x: 0, y: 0, from: { x, y } } );
-
-		return () => {
-			controller.stop();
-		};
```

Comment:
This bug was introduced in https://github.com/WordPress/gutenberg/pull/57133/files#diff-df7dd6cfc5ba58aa8a6e243aed0268419c7bcf518307f637f2989c522395e086R143-R145 as part of the WP 6.5 cycle.